.github/workflows/getsentry-dispatch.yml

```diff
@@ -8,14 +8,38 @@ on:
   #
   # See https://github.com/getsentry/sentry/pull/21600 for more details
   pull_request_target:
+    types: [labeled, opened, reopened, synchronize]
+
+# disable all other special privileges
+permissions:
+  # needed for `actions/checkout` to clone the code
+  contents: read
+  # needed to remove the pull-request label
+  pull-requests: write
 
 jobs:
   dispatch:
+    if: github.event.action != 'labeled' || github.event.label.name == 'external-trigger-getsentry'
```

Comment:
this is necessary so unrelated labels don't trigger this workflow.  it has the (unfortunate) side-effect of making this become [skipped] (potentially hiding the error message below) -- I haven't come up with a good way to avoid that yet -- might have to resort to (annoying) comments T.T -- will follow up with that in a v2 since I'd like to get this unblocked

Reply:
oh another idea would be to have it leave a separate commit status via the getsantry app

Reply:
I don't want to make the experience worse for internal employees (needing the label)